src/parser/formatting-evaluator-module.ts

```diff
@@ -41,11 +41,17 @@ export class FormattingEvaluatorModule implements Module {
       this._multipliers = this._configuration.multipliers.reduce((acc, curr) => {
         return {
           ...acc,
-          [curr.role.reduce((a, b) => this._getEnumValue(b) | a, 0)]: {
```

Comment:
@0x4007 This is the fix for the tags. Basically when using a list of `roles` the enum was not properly deduced (still using the old enum combination with `ISSUE | AUTHOR` instead of `ISSUE_AUTHOR`

Reply:
I included the length limit because it was a small change that was needed as well, could have split the PR sorry.